### ISSUE TITLE
[Mention The Feature]: text animation is needed at landing page to the main text..

### ISSUE THREAD

**Issue by 2004pra** [ML-Fusion-Lab/ML-Fusion-Lab-Website#994]
### Is your feature request related to a problem? Please describe.

--> i would like to provide text animation to your website home page
the text is "Welcome to rentalog"
-->with some major css changes and adding a few file it will get the perfect animation

### Describe the solution you'd like.

by adding some files and adding the animation over it
using the css
-->while some one visit the website the normal text does not look great and intuitive.

### Describe alternatives you've considered.

i have total 3 types of animation in my mind not any alternatives but multiple options for betterment and matching the theme of the website ..

### Additional context.

please assign this issue under gssoc extd 24 , hactoberfest, level2

### Show us the magic with screenshots

THIS IS HOW IT LOOKS LIKE RIGHT NOW BIT I WILL CHANGE IT SOON..
![Screenshot 2024-10-24 013535](https://github.com/user-attachments/assets/39260ddb-d0e8-484e-9d86-fbdc77cec9ca)

### Checklist

- [X] I have checked the [Existing Issues](https://github.com/ML-Fusion-Lab/ML-Fusion-Lab-Website/issues)
- [X] I have read the [Contributing Guidelines](https://github.com/ML-Fusion-Lab/ML-Fusion-Lab-Website/blob/main/contributing.md)
- [X] I want to work on this issue. (optional)

**Comment by vivekvardhan2810** [comment]
for now we are not looking for this.

so i am closing this issue.

please understand.

**Comment by 2004pra** [comment]
please i didnt got any issue this week brother

**Comment by vivekvardhan2810** [comment]
i can give you an issue @2004pra but you have to complete it fast.

will it be okay for you.

**Comment by vivekvardhan2810** [comment]
And @2004pra one more thing this project won't be active from october 25th, so you have to complete before that date.

can you do it.

**Comment by 2004pra** [comment]
yup it will be ready by now

**Comment by vivekvardhan2810** [comment]
you can create an issue like adding a scroll up button in testimonial page.

Can you do this @2004pra

**Comment by 2004pra** [comment]
ya but i also want to do this one

**Comment by vivekvardhan2810** [comment]
but i don't want this for now, instead of this, i can give you this.

> you can create an new issue like adding a scroll up button in testimonial page.
> 
> Can you do this @2004pra

**Comment by 2004pra** [comment]
ok 
i am rasising the scroll up button issue

**Comment by vivekvardhan2810** [comment]
@2004pra keep the issue title as adding a scroll up button in testimonial page.

**Comment by 2004pra** [comment]
every page has scroll which one you are saying about  feedback page also has scroll up button

**Comment by vivekvardhan2810** [comment]
i am saying about testimonial page.

**Comment by 2004pra** [comment]
share the screenshot

**Comment by vivekvardhan2810** [comment]
in home page, click the community support button.

then there you can see testimonial button.

![image](https://github.com/user-attachments/assets/ef47f710-8ccf-495b-812a-9d920d125b7b)

then click the testimonial button.

then you can see the testimonial page.

**Comment by 2004pra** [comment]
thanks sir

**Comment by vivekvardhan2810** [comment]
Can you see the testimonial page @2004pra
